packages/connect/CHANGELOG.md

```diff
@@ -1,23 +1,21 @@
 |             Package              | Stable |    Canary    |
 | :------------------------------: | :----: | :----------: |
-|       npm @trezor/connect        | 9.4.7  | 9.4.8-beta.2 |
-|     npm @trezor/connect-web      | 9.4.7  | 9.4.8-beta.2 |
-| npm @trezor/connect-webextension | 9.4.7  | 9.4.8-beta.2 |
+|       npm @trezor/connect        | 9.4.7  | 9.5.0-beta.1 |
+|     npm @trezor/connect-web      | 9.4.7  | 9.5.0-beta.1 |
+| npm @trezor/connect-webextension | 9.4.7  | 9.5.0-beta.1 |
 |    npm @trezor/connect-mobile    |   -    | 0.0.1-beta.1 |
 
 |     Deployment     | Stable |    Canary    |
 | :----------------: | :----: | :----------: |
-| connect.trezor.io/ | 9.4.7  | 9.4.8-beta.2 |
+| connect.trezor.io/ | 9.4.7  | 9.5.0-beta.1 |
 
-Use the persistent link [connect.trezor.io/9](https://connect.trezor.io/9/) to access the latest stable version of Connect Explorer.
-
-# 9.4.8-beta.2
+| connect.trezor.io/ | 9.4.7 | 9.4.8-beta.2 |
```

Comment:
this line is indeed duplicated. something wrong with the script

Reply:
Interesting, I will have a look  at the script.